lib/helper/Nightmare.js

```diff
@@ -156,7 +157,30 @@ class Nightmare extends Helper {
     });
   }
 
+  _beforeSuite() {
+    if (!this.options.restart)
+      this._startBrowser();
```

Comment:
Creates new instance, is restart is falsy. Will not impact existing functionality.